scripts/prerelease/generate_prerelease_script.py

```diff
@@ -163,6 +163,11 @@ def beforeInclude(self, *args, **kwargs):
 
     # use random source repo to pass to devel job template
     source_repository = deepcopy(list(repositories.values())[0])
+    if not source_repository:
+        print(('The repository \'%s\' does not have a source entry in the distribution ' +
```

Comment:
Please keep double quotes for this line in order to not require escaping for the single quotes inside the string (PEP8).